### PR TITLE
refactor(bigquery): update code samples to use strings for table and dataset IDs

### PR DESCRIPTION
**Pull request by Emar-Kar**
List of changes:
-
### **New samples:**
1.   client_query_add_column;
2.   client_query_batch;
3.   client_query_destination_table;
4.   client_query_destination_table_cmek;
5.   client_query_destination_table_legacy;
6.   client_query_dry_run;
7.   client_query_legacy_sql;
8.   client_query_relax_column;
9.   copy_table_cmek;
10. copy_table_multiple_source.
### **Comment lines in samples unified**:
1. client_query;
2. copy_table;
3. load_table_dataframe.
### **DEPRECATED tests**
1. test_create_routine;
2. test_update_routine;
3. test_get_routine;
4. test_delete_routine;
5. test_list_routines;
6. test_create_routine_ddl

**All those tests were united in the `test_routine_samples` file!**

### CODE REVIEW

File: bigquery/samples/client_query_destination_table_cmek.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    # TODO(developer): Set table_id to the ID of the destination table.
+    # table_id = "your-project.your_dataset.your_table_name"
+
+    job_config = bigquery.QueryJobConfig()
+    job_config.destination = table_id
+
+    # Set the encryption key to use for the destination.
+    # TODO(developer): Replace this key with a key you have created in KMS.
+    kms_key_name = "projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}".format(
+        "cloud-samples-tests", "us-central1", "test", "test"
```
Comment (HemangChothani) [inline bigquery/samples/client_query_destination_table_cmek.py]: I think you need to change location "us-central1" to "us", I have received same comment on unit-test and also see git issue [9296] , in that @tswast describe.

File: bigquery/samples/copy_table_cmek.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    # TODO(developer): Set dest_table_id to the ID of the destination table.
+    # dest_table_id = "your-project.your_dataset.your_table_name"
+
+    # TODO(developer): Set orig_table_id to the ID of the original table.
+    # orig_table_id = "your-project.your_dataset.your_table_name"
+
+    # Set the encryption key to use for the destination.
+    # TODO(developer): Replace this key with a key you have created in KMS.
+    kms_key_name = "projects/{}/locations/{}/keyRings/{}/cryptoKeys/{}".format(
+        "cloud-samples-tests", "us-central1", "test", "test"
```
Comment (HemangChothani) [inline bigquery/samples/copy_table_cmek.py]: same as above.

File: bigquery/samples/client_query_destination_table_cmek.py (hunk 1 above)
Comment (Emar-Kar) [reply bigquery/samples/client_query_destination_table_cmek.py]: I saw this issue before, but can you tell me exactly where did you receive this comment?

File: bigquery/samples/client_query_destination_table_cmek.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    # Start the query, passing in the extra configuration.
+    query_job = client.query(
+        "SELECT 17 AS my_col;",
+        location="US",  # Must match the source and the destination dataset(s) location.
+        job_config=job_config,
+    )  # Make an API request.
+    query_job.result()  # Wait for the job to complete.
+
+    table = client.get_table(table_id)  # Make an API request.
+    if table.encryption_configuration.kms_key_name == kms_key_name:
```
Comment (HemangChothani) [inline bigquery/samples/client_query_destination_table_cmek.py]: Just from information  `EncryptionConfiguration` class moved from table to an independent file , but  it's still support to import from table.py file as backward compatibility  .

File: bigquery/samples/client_query_destination_table_cmek.py (hunk 1 above)
Comment (HemangChothani) [reply bigquery/samples/client_query_destination_table_cmek.py]: please see my PR https://github.com/googleapis/google-cloud-python/pull/9302#discussion_r328843331

File: bigquery/samples/client_query_destination_table_cmek.py (hunk 3 above)
Comment (Emar-Kar) [reply bigquery/samples/client_query_destination_table_cmek.py]: This is the **property** of the `Table` class.

File: bigquery/samples/client_query_destination_table_cmek.py (hunk 1 above)
Comment (Emar-Kar) [reply bigquery/samples/client_query_destination_table_cmek.py]: Yeah, thanks. I'll write to Tim to get to know for sure.

File: bigquery/samples/client_query_destination_table_cmek.py (hunk 3 above)
Comment (HemangChothani) [reply bigquery/samples/client_query_destination_table_cmek.py]: oh yes correct! my bad , i just posted for info.

**Pull request closed without merging**